packages/README.md

```diff
@@ -0,0 +1,107 @@
+# Assertive.ts Sinon
+
+An Assertive.ts plugin to match over [Sinon.js](https://sinonjs.org/) spies, stubs, mocks, and fakes.
+
+## Requirements
+
+- **@assertive-ts/core:** >=2.0.0
+- **sinon:** >=15.2.0
+
+## Install
+
+```sh
+npm install --save-dev @assertive-ts/sinon
+```
+
+Or:
+
+```sh
+yarn add --dev @assertive-ts/sinon
+```
+
+## API Reference
+
+You can find the full API reference [here](https://stackbuilders.github.io/assertive-ts/docs/sinon/build/) 📚
+
+## Usage
+
+You just need to load the plugin in a file that runs before the tests execution, like a `setup.ts` or something like that:
+
+```ts
+// setup.ts
+
+import { usePlugin } from "@assertive-ts/core";
+import { SinonPlugin } from "@assertive-ts/sinon";
+
+usePlugin(SinonPlugin);
+
+// ...rest of your setup
+```
+
+And that's it! The extra types will be automatically added as well so you won't need to change anything on TypeScript's side. Now you can use the `expect(..)` function to assert over Sinon spies, stubs, mocks, and fakes.
+
+```ts
+import { expect } from "@assertive-ts/core";
+import Sinon from "sinon";
+
+const spy = Sinon.spy(launchRockets);
+
+expect(spy).toBeCalled(3); // exactly 3 times
+
+expect(spy).toBeCalledTwice();
+
+expect(spy).toBeCalledAtLeast(2);
+
+expect(spy).toBeCalledAtMost(3);
+
+expect(spy).toHaveArgs(10, "long-range");
+
+expect(spy).toThrow();
+```
+
+The assertion above act over any of the calls made to the spy. You can get more specific matchers if you assert over a single spy call:
```

Comment:
```suggestion
The assertion above acts over any of the calls made to the spy. You can get more specific matchers if you assert over a single spy call:
```

Reply:
In this case, `act` is the correct word because the phrase is actually `The assertions above act over...` 🙂